dozer-k8/dozer_data/dozer-config.yaml

```diff
@@ -0,0 +1,47 @@
+app_name: dozer-taxi-apis
```

Comment:
These are supposed to be not included. We expected users to provide their own `dozer-config.yaml`

Reply:
Can you remove them from the PR ?

Reply:
Do you want me to remove dozer-config.yaml altogether?

Reply:
yes.

Reply:
Done.